linode/manager
# M3-5663: Changes to Upgrade to High Availability Dialog Copy

## Description

Changes the copy for the Upgrade to High Availability Dialog to reflect the impact it will have on existing clusters and their availability before, during, and after the upgrade.

## How to test

Navigate to a Kubernetes Cluster that can be upgraded to a HA cluster.
Click the Upgrade to HA Cluster button
Ensure the new copy matches the mock-up
